packages/opentelemetry-exporter-collector-grpc/src/util.ts

```diff
@@ -108,8 +108,11 @@ export function send<ExportItem, ServiceRequest>(
 }
 
 export function validateAndNormalizeUrl(url: string): string {
+  if (!url.match(/:\/\//)) {
```

Comment:
While this will generally work, it's probably a little simplistic as it would be possible for someone to pass this function a url which doesn't have a protocol but does include ```://``` somewhere in the url (assuming the ```:``` was not encoded as %3A.

Probably really should check that it "starts" with a protocol using something like the following to identify a leading protocol between 1 and 8 characters in length
```/^([\w]{1,8}):\/\//```

Reply:
sigh why is input validation so hard :(

Reply:
applied.